lib/components/app/locale-selector.tsx

```diff
@@ -44,31 +44,42 @@ const LocaleSelector = (props: LocaleSelectorProps): JSX.Element => {
   }
 
   return (
-    <NavDropdown
+    <Dropdown
       id="locale-selector"
-      pullRight
+      // TODO: How to make this work without block ruby?
+      style={{ display: 'block ruby' }}
       title={
-        <StyledIconWrapper style={{ color: 'rgba(255, 255, 255, 0.85)' }}>
+        <span
+          style={{
+            color: 'rgba(255, 255, 255, 0.85)'
+          }}
+        >
           <GlobeAmericas />
```

Comment:
Set the height of this SVG image to 18px to prevent the header bar from bleeding on the search form.